app/pages/notifications.cjsx

```diff
@@ -0,0 +1,145 @@
+React = require 'react'
+Loading = require '../components/loading-indicator'
+Paginator = require '../talk/lib/paginator'
+ChangeListener = require '../components/change-listener'
+talkClient = require '../api/talk'
+Notification = require './notifications/notification'
+
+module?.exports = React.createClass
+  displayName: 'NotificationsPage'
+
+  propTypes:
+    project: React.PropTypes.object
+    user: React.PropTypes.object
+
+  getDefaultProps: ->
+    query: page: 1
+
+  getInitialState: ->
+    firstMeta: { }
```

Comment:
Tracks meta for the top and bottom paginator separately.

This lets us increase the bottom page and top page independently.